umm-spec-lib/src/cmr/umm_spec/migration/related_url_migration.clj

```diff
@@ -91,10 +91,24 @@
    Need to remove MimeType, add default for Name and Description if they don't exist."
   [element]
   (if-let [ol-resource (:OnlineResource element)]
-    (-> element
-        (assoc :OnlineResource {:Linkage (:Linkage ol-resource)
-                                :Name (util/with-default (:Name ol-resource) true)
-                                :Description (util/with-default (:Description ol-resource) true)}))
+    (assoc element :OnlineResource {:Linkage (:Linkage ol-resource)
+                                    :Name (util/with-default (:Name ol-resource) true)
+                                    :Description (util/with-default (:Description ol-resource) true)})
+    element))
+
+(defn- remove-default-name-description
+  "Remove default Name and Description from online-resource."
```

Comment:
Try to add to this docstring something about why we want to remove defaults in this case. This is one thing that we're often inconsistent about. I think we should be better about documenting why that is, even if that's just "UMM 1.x doesn't have defaults, but UMM 1.y does"

Reply:
We don't have to remove these default values for the code to function correctly in the real world. It just looks cleaner when we add default values migrating one direction and remove the default values when migrating the opposite direction. This is why it worked fine without this direction of migration in my previous PR.  I think Adding simple comment is just going to be more confusing.

Reply:
Ordinarily, I'd want to see a tiny bit more in the docstring for a utility function, but this is only used once right now, and in this same file. The split was really made just to keep the code clean and simple, and it's not a public function or anything.

We will want to revisit this when another function use the utility function here, but that additional use can be used to improve the docs at that time. I'm okay with this as it is.